index.ts

```diff
@@ -41,7 +42,7 @@ const SeverityAnnotationLevelMap = new Map<RuleSeverity, "warning" | "failure">(
     owner: ctx.repo.owner,
     repo: ctx.repo.repo,
     name: CHECK_NAME,
-    head_sha: ctx.sha,
+    head_sha: ctx.payload.pull_request ? ctx.payload.pull_request.head.sha : ctx.sha,
```

Comment:
Why did you reference head.sha?

Reply:
Because if you use `ctx.sha` - it won't appear in the PR: https://github.com/actions/toolkit/issues/133#issuecomment-532832187